package.json

```diff
@@ -1,62 +1,62 @@
 {
- "name": "react-transform-boilerplate",
- "version": "1.0.0",
- "description": "A new Webpack boilerplate with hot reloading React components, and error handling on module and component level.",
- "scripts": {
-   "clean": "rimraf dist",
-   "build:webpack": "cross-env NODE_ENV=production webpack --config webpack.config.prod.js",
-   "build": "npm run clean && npm run build:webpack",
-   "start": "node devServer.js",
-   "lint": "eslint src"
- },
- "repository": {
-   "type": "git",
-   "url": "https://github.com/gaearon/react-transform-boilerplate.git"
- },
- "keywords": [
-   "react",
-   "reactjs",
-   "boilerplate",
-   "hot",
-   "reload",
-   "hmr",
-   "live",
-   "edit",
-   "webpack",
-   "babel",
-   "react-transform"
- ],
- "author": "Dan Abramov <dan.abramov@me.com> (http://github.com/gaearon)",
- "license": "CC0-1.0",
- "bugs": {
-   "url": "https://github.com/gaearon/react-transform-boilerplate/issues"
- },
- "homepage": "https://github.com/gaearon/react-transform-boilerplate",
- "devDependencies": {
-   "babel-core": "6.10.4",
-   "babel-eslint": "5.0.4",
-   "babel-loader": "6.2.4",
-   "babel-preset-es2015": "6.9.0",
-   "babel-preset-react": "6.11.1",
-   "babel-preset-react-hmre": "1.1.1",
-   "cross-env": "1.0.8",
-   "eslint": "1.10.3",
-   "eslint-plugin-babel": "3.3.0",
-   "eslint-plugin-react": "3.16.1",
-   "eventsource-polyfill": "0.9.6",
-   "express": "^4.13.3",
-   "rimraf": "^2.4.3",
-   "webpack": "1.13.1",
-   "webpack-dev-middleware": "1.6.1",
-   "webpack-hot-middleware": "2.12.1"
- },
- "dependencies": {
-   "jquery": "^3.0.0",
-   "lodash": "^4.13.1",
-   "material-ui": "^0.15.1",
-   "react": "^15.2.0",
-   "react-dom": "^15.2.0",
-   "react-router": "^2.5.2",
-   "react-tap-event-plugin": "^1.0.0"
- }
+  "name": "react-transform-boilerplate",
```

Comment:
This is a weird checkin. Did you change anything in this file? Did you add a library or something? If not, you should not have a change in this file and definitely not commit a change like this.